flight/Modules/Attitude/acro/attitude.c

```diff
@@ -199,7 +198,6 @@ static void AttitudeTask(void *parameters)
 			// For first 7 seconds use accels to get gyro bias
 			accelKp = 0.1f + 0.1f * (PIOS_Thread_Systime() < 4000);
 			accelKi = 0.1;
-			yawBiasRate = 0.1;
```

Comment:
Sorry, not this one, the next one.

We set yawbiasrate to 0.1 to try and more aggressively set the yaw bias to 0 initially.

At the same time, we're accumulating gyro samples to set an offset to make them average to 0 at the arming edge.